src/FieldWarning/Assets/UI/Ingame/UnitLabel/UnitLabelAttacher.cs

```diff
@@ -9,57 +9,32 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is
  * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
  * the License for the specific language governing permissions and limitations under the License.
+ *
  */
 
-using UnityEngine;
-
-public class UnitLabelAttacher : MonoBehaviour
+namespace Assets.UI.Ingame.UnitLabel
 {
-    public GameObject Label;
-    private Canvas _canvas;
-    private Collider _collider;
-    private Vector3 _paddingVector;
+    using UnityEngine;
 
-    public void Start()
+    public class UnitLabelAttacher : MonoBehaviour
     {
-        _canvas = GameObject.Find("UIWrapper").GetComponent<Canvas>();
-        _collider = GetComponentInChildren<Collider>();
+        public GameObject Label { get; private set; }
 
-        Label = Instantiate(Resources.Load<GameObject>("UnitLabel"), _canvas.transform);
+        public bool Visible
+        {
+            get => Visible;
+            set => Label.SetActive(value);
+        }
 
-        _paddingVector = new Vector3(0, 5, 0);
-    }
-
-    public void LateUpdate()
-    {
-        Label.transform.position = GetScreenPosition(_canvas, Camera.main);
-    }
-
-    public void Hide()
-    {
-        Label.SetActive(false);
-    }
-
-    public void Show()
-    {
-        Label.SetActive(true);
-    }
-
-    public void SetVisibility(bool visible)
-    {
-        if (visible)
-            Show();
-        else
-            Hide();
-    }
-
-    public Vector3 GetScreenPosition(Canvas canvas, Camera cam = null)
-    {
-        if (cam == null)
-            cam = Camera.main;
+        public Vector3 GetScreenPosition() => Camera.main.WorldToScreenPoint(transform.position + new Vector3(0f, 5f));
 
-        var labelPos = cam.WorldToScreenPoint(transform.position + _paddingVector);
+        public void LateUpdate() => Label.transform.position = GetScreenPosition();
 
-        return labelPos;
+        public void Start()
+        {
+            Label = Instantiate(
+                Resources.Load<GameObject>("UnitLabel"),
+                GameObject.Find("UIWrapper").GetComponent<Canvas>().transform);
```

Comment:
When breaking up an expression I like to use double indent to distinguish it from regular control flow indentation. It's not required because you can't make VS do it automatically without a plugin, just mentioning it in case you like it.